### DISCUSSION TITLE
[AzureDevOps] How to Fail the Pipeline if Error/Problem

### DISCUSSION THREAD

**Discussion by lfaltran** [JetBrains/Qodana#160] [Q&A] [answered]
Hi, I'm looking to set my pipeline as FAILED on Azure Dev Ops if some Error/Problem was notified by Qodana.

There is any way to configure this behavior?

**Comment by brichbash** [comment, marked as answer]
Hello,

Adding `--fail-threshold, <number-of-problems>` to the `args` option of the Qodana task should fail your build once the number of problem exceeds the set threshold.

**Comment by lfaltran** [comment]
Thank you @brichbash this is exactly what was I looking for.

How I was working with YML file, the result looks like below:

```version: "1.0"
linter: jetbrains/qodana-jvm-community:2022.3
profile:
  name: qodana.recommended
failThreshold: 10
